irods/client_configuration/__init__.py

```diff
@@ -147,17 +158,15 @@ def loadlines(entries, common_root = sys.modules[__name__]):
             _load_config_line(eval_func = identity, **e_)
 
 
-# TODO remove this test
-def test_it():
-    with loadlines(entries=(
-        dict(setting='data_objects.auto_close',value=True),
-    )):
-        print('dac (during) =',data_objects.auto_close)
-        pass
-    print('dac (after) =',data_objects.auto_close)
-    1
+def _load_config_line(root, setting, value, return_old = None, eval_func = ast.literal_eval):
+    """Low-level utility function for loading a line of settings, with the option to return the old (displaced) value.
+
+       The 'root' refers to the starting point in the configuration tree.  Its meaning is the same as in loadlines().
+       The 'setting' is a string containing the dotted name for the configuration setting.
+       The 'value' is the new value to be loaded.  This will be evaluated via 'eval_func'.
+       The 'old_value' is either None or a list which returns the displaced value back to the caller.
```

Comment:
Should this say `return_old`? Also, missing documentation for `eval_func`. Also-also, bonus points for making this look more like `loadlines`'s docstring for consistency.

Reply:
corrected.